.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
```

Comment:
@athrael-soju do u want to keep semi?

saw this in the their docs:

https://prettier.io/docs/en/configuration.html#basic-configuration


and thought to add since it's opinionated :D


https://prettier.io/docs/en/rationale